lib/bgs/vnp_veteran.rb

```diff
@@ -29,19 +29,13 @@ def create
           va_file_number: @va_file_number,
           claim_type_end_product: claim_type_end_product,
           location_id: location_id,
-          net_worth_over_limit_ind: net_worth_over_limit_ind
+          net_worth_over_limit_ind: veteran.formatted_boolean(@payload['dependents_application']['household_income'])
```

Comment:
Should this be veteran.formatted_boolean(@payload['dependents_application']['household_income'])
or should it just be formatted_boolean(@payload['dependents_application']['household_income'])?

Reply:
@thilton-oddball yes, It should be `veteran.formatted_boolean` because the `formatted_boolean` method is in the `models/bgs_dependents/base.rb` Base class, and the veteran model inherits from that class